tests/test_plotting.py

```diff
@@ -569,6 +569,45 @@ def test_vsel_iterable(mapdl, make_block):
     )
 
 
+def test_color_areas(mapdl, make_block):
+    pl = mapdl.aplot(vtk=True, color_areas=True, return_plotter=True)
```

Comment:
```suggestion
    mapdl.aplot(vtk=True, color_areas=True, return_plotter=True)
```

Reply:
This is required by [coday](https://app.codacy.com/gh/ansys/pymapdl/pullRequest?prid=12246187)